examples/run_config.py

```diff
@@ -0,0 +1,253 @@
+# -*- coding: utf-8 -*-
+"""
+Using :class:`~caf.toolkit.BaseConfig`
+======================================
+
+The :class:`~caf.toolkit.BaseConfig` class in caf.toolkit is designed to load
+and validated YAML configuration files. This example shows how
+to create child classes of BaseConfig to load parameters.
+
+See Also
+--------
+:class:`~caf.toolkit.arguments.ModelArguments`
+    for producing command-line arguments from a BaseConfig class.
+"""
+
+# Built-Ins
+import pathlib
+import re
+from typing import Any, Self
+
+# Third Party
+import pydantic
+from pydantic import dataclasses
+
+# Local Imports
+import caf.toolkit as ctk
+
+folder = pathlib.Path().parent
+
```

Comment:
The way this renders it looks like this code is part of the "See Also" section. I think this is just setup to make the rest of the code work? 

Should we add a heading to make it clear that this is just setup for the rest of the code?